.github/actions/paths-filter/action.yml

```diff
@@ -98,3 +107,12 @@ runs:
       filters: |
         identity-frontend-change:
           - 'identity/client/**'
+
+  - uses: dorny/paths-filter@v3
+    id: filter-zeebe
+    with:
+      base: ${{ github.event.merge_group.base_ref || '' }}
+      ref: ${{ github.event.merge_group.head_ref || github.ref }}
+      filters: |
+        zeebe-change:
+          - 'zeebe/**'
```

Comment:
changes in dist/ common/ clients/ etc. should also trigger Zeebe CI.

Reply:
Thanks. 👍🏼  Dist is part of

Reply:
What about the identiy, search and service modules?

Reply:
Integration tests are part of the common ci. Or do you mean smoke tests?

Reply:
> What about the identiy, search and service modules?

Not sure what you mean. You we should run Zeebe ci when identity changes?

Reply:
> run Zeebe ci when identity changes

We have a dependency on identity so yes we should run Zeebe CI, otherwise it can break when Identity makes changes.

> Integration tests are part of the common ci. Or do you mean smoke tests?

Ah okay. It's not clear to me what runs where but looks like you are correct and smoke tests are missing.

Reply:
Discussed with @npepinpe. Current state, identity is not really integrated into the mono-repo. 

Zeebe is using still the old identity-spring-boot-starter https://github.com/camunda/camunda/blob/main/parent/pom.xml#L980-L984